source/sdk/IO.ooc

```diff
@@ -6,7 +6,7 @@
  * of the MIT license.  See the LICENSE file for details.
  */
 
-include stdio, fcntl, unistd
+include stdio | (_POSIX_SOURCE), fcntl, unistd
```

Comment:
I think the windows build is currently broken even without it :( I have a plan to build and run on windows this week though